src/sensors/can/C620.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef C620_DRIVER_HPP
+#define C620_DRIVER_HPP
+
+#include "motor.hpp"
+
+// C620 Data Sheet
+// https://rm-static.djicdn.com/tem/17348/RoboMaster%20C620%20Brushless%20DC%20Motor%20Speed%20Controller%20V1.01.pdf
+
+/// @brief C620 controller driver. This manages generating CAN output messages and processing incomming CAN messages into a state array.
+/// @note It's construction is heavily managed since copying this object could alter the actions of CAN (and by extension the robot). This object exists only to be managed by CANManager.
```

Comment:
see other review comment, allow motors to be copied. Explain the issue with CANManager (which is not an issue with motors)